sdk/core/src/lib.rs

```diff
@@ -57,6 +57,13 @@ pub use response::*;
 pub use seekable_stream::*;
 pub use sleep::sleep;
 
+// re-export packages
+pub use http;
+pub use url;
```

Comment:
Why? It would be really nice not to expose these entire crates. This means the core crate's semver story is directly tied to two other crates.

Reply:
I'm okay with that. I added this given previous feedback where @bmc-msft preferred whole packages over individual types. `form_urlencoded` is also needed. It is a [re-export from Url](https://docs.rs/url/latest/url/#reexports).

Reply:
I removed the package exports to get this PR merged.